src/MassTransit/MassTransit.csproj

```diff
@@ -1,47 +1,48 @@
-    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
-    <PackageReference Include="System.Xml.ReaderWriter" Version="4.3.0" />
-    <PackageReference Include="System.Threading.ThreadPool" Version="4.3.0" />
-    <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
-    <Reference Include="System.Transactions" />
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Monitoring\Performance\Configuration" />
-  </ItemGroup>
+    <PropertyGroup>
+        <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\..\MassTransit.snk</AssemblyOriginatorKeyFile>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+        <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
+    </PropertyGroup>
+    <PropertyGroup>
+        <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\MassTransit.xml</DocumentationFile>
+    </PropertyGroup>
+    <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+        <Compile Remove="Util/NHProfIntegration.cs" />
+        <Compile Remove="Configuration/BusConfigurators/PerformanceCounterBusFactorySpecification.cs" />
+        <Compile Remove="Serialization/StaticHeaders.cs" />
+        <Compile Remove="Serialization/StaticConsumeContext.cs" />
+        <Compile Remove="Serialization/ExtensionsForBinaryMessageSerializer.cs" />
+        <Compile Remove="Serialization/BinaryMessageSerializer.cs" />
+        <Compile Remove="Serialization/BinaryMessageDeserializer.cs" />
+        <Compile Remove="Monitoring/Performance/**" />
+        <Compile Remove="Configuration/PerformanceCounterExtensions.cs" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="GreenPipes" Version="2.1.3" />
+        <PackageReference Include="NewId" Version="3.0.1" />
+        <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+        <PackageReference Include="Newtonsoft.Json.Bson" Version="1.0.1" />
+        <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
+        <PackageReference Include="System.ValueTuple" Version="4.4.0" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' ">
+        <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
+        <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
+        <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
+        <PackageReference Include="System.Xml.ReaderWriter" Version="4.3.0" />
+        <PackageReference Include="System.Threading.ThreadPool" Version="4.3.0" />
+        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.5.1" />

```

Comment:
Can you change this to v4.3.0 to match the others?

Reply:
it couldn't be used, Activity do not exists in v4.3.0, I've tried

Reply:
Hmm. Okay. I just pushed a new version, curious if it will build if you rebase against develop since there shouldn't be a package downgrade.

Reply:
Marten was the conflicting package, and I removed all those explicit dependencies.

Reply:
how can I do this?

Reply:
merged